packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1207,6 +1218,17 @@
                 }
             ]
         },
+        {
+            "name": "session_start",
```

Comment:
Why is this needed? There is already a metric for starting the plugin.

Reply:
What is the metric?

Reply:
Yeah, I plan to do this. It was just due to time constraints. I'll follow up after this PR